.github/PULL_REQUEST_TEMPLATE/dependency_updates_template.md

```diff
@@ -12,6 +12,7 @@ These dependencies have been updated to their latest versions:
 - `entities`
 - `eslint-plugin-react`
 - `eslint`
+- `eslint-plugin-react`
```

Comment:
Ope. I think I meant to move this to alpha order, not duplicate. Thanks! I'll just remove it for now and move it later since I need to review all the packages there.